src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -170,10 +170,14 @@ else if(loans.getTotalRecords() == 1) {
     return result.combineAfter(userRepository::getUser, Loan::withUser);
   }
 
-  public CompletableFuture<Result<MultipleRecords<Loan>>> findClosedLoans(
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findLoansToAnonymize(
     PageLimit pageLimit) {
 
-    return queryLoanStorage(getStatusCQLQuery("Closed"), pageLimit);
+    // Are closed and not yet anonymized.
+    Result<CqlQuery> cqlQuery = getStatusCQLQuery("Closed")
+      .combine(CqlQuery.match("userId", ""), CqlQuery::and);
```

Comment:
What CQL does `match`  generate, at first glance this suggests it is trying to find loans with a blank `userId`?

Or is this a quirk of CQL, and that an empty string means match any value, except for no value at all?

Reply:
As per RMB documentation the `userId=""` should match every record where the userId property is defined (i.e. non null).

Reply:
Maybe a new method could be introduced to hide the details of the query and reveal the intent better. Something like `present` for a query which checks that an index is present, or if `present` isn't clear, maybe `hasValue` as an alternative?

Reply:
Renamed to `hasValue`.